crates/autopilot/src/domain/eth/mod.rs

```diff
@@ -0,0 +1,25 @@
+pub use {
```

Comment:
Why is this part of the domain?

Reply:
Very good question. I assumed these were used somewhere in the domain of the `driver` but I can't find it. Essentially it's only referenced from `infra` so no reason to keep it in the `domain`.

Reply:
moved.